Automattic/jetpack
# VideoPress: show image when the block acts as an example

This PR shows a static image when the block acts as an example

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: show a static image when the block acts as an example

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block editor
* Open the block inserter
* Mouse over the VideoPress video block
* Confirm the block preview shows the VP image

before | after
------|------
<img width="316" alt="image" src="https://user-images.githubusercontent.com/77539/204883317-86a63510-a326-4a91-9890-d0e7af9d46d4.png"> | <img width="316" alt="image" src="https://user-images.githubusercontent.com/77539/204883092-4e622370-2094-465f-b272-a58c5f9cfc01.png">
